New entries in UI and UX -- fixing PR (#4)
js6450/docs: content/uiux/concepts/convergent-thinking/convergent-thinking.md

---
Title: 'Convergent Thinking'
Description: 'Convergent thinking is a creative mode of thinking that analyzes, filters, and focuses ideas to make decisions.'
Subjects:
  - 'Web Design'
Tags:
  - 'UI'
  - 'UX'
CatalogContent:
  - 'intro-to-ui-ux'
  - 'paths/front-end-engineer-career-path'
---

**Convergent thinking** is a creative mode of thinking that analyzes, filters, and focuses ideas to make decisions.

Convergent thinking involves grouping and sorting ideas generated through [divergent thinking](https://www.codecademy.com/resources/docs/uiux/divergent-thinking) to reveal themes and evaluate concepts through techniques like [affinity diagrams](https://www.usertesting.com/blog/affinity-mapping). It's possible and encouraged to move between both convergent and divergent thinking. If you are trying to converge on a solution but don't feel you have enough information, it might be time to return to a divergent mode of thinking and gather more data before moving forward.

> Many design methodologies explicitly define stages where divergent thinking is the focus. For example, in the double diamond diagram, the "Define" stage focuses on convergent strategy, or sorting and analyzing information gathered during discovery, and the "Deliver" stage focuses on convergent execution, or developing and delivering a solution.

Reviewer: ```suggestion
Many design methodologies explicitly define stages where convergent thinking is the focus. For example, in the double diamond diagram, the "Define" stage focuses on convergent strategy, or sorting and analyzing information gathered during discovery, and the "Deliver" stage focuses on convergent execution, or developing and delivering a solution.
```

> ![The Double Diamond diagram: two diamonds side-by-side, broken down into Strategy (Discover and Define) and Execution (Develop and Deliver) phases, moving from problem to solution.](https://static-assets.codecademy.com/Courses/intro-to-ui-and-ux/key-methodologies/double-diamond.png)

Reviewer 2: Same thing here. Can we relocate the image to the **media/** directory and make the following change?

```suggestion
![The Double Diamond diagram: two diamonds side-by-side, broken down into Strategy (Discover and Define) and Execution (Develop and Deliver) phases, moving from problem to solution.](https://raw.githubusercontent.com/Codecademy/docs/main/media/double-diamond.png)
```

Reviewer 3: Resolving per [this](https://github.com/Codecademy/docs/commit/85b3f4adf4015d22769280fe2db1852c51d848d9) inclusion to `content-standards.md`

> For more information, visit the [Interaction Design Foundation's convergent thinking definition](https://www.interaction-design.org/literature/topics/convergent-thinking).

Reviewer 2: For Docs entries, we want to employ a voice that manual-like but inviting/engaging. (without "you/we" pronouns, directives, or imperative verbs) What do we think about the following suggestion?

```suggestion
More information can be found in the [Interaction Design Foundation's convergent thinking definition](https://www.interaction-design.org/literature/topics/convergent-thinking).
```
